search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -110,9 +110,10 @@
    :temporal (conj exclude-plus-and-or-option :limit-to-granules)
    :revision-date cpv/and-option
    :created-at cpv/and-option
+   :granule-created-at cpv/and-option
```

Comment:
Is this working as expected? I thought the parameter was `has-granules-created-at`.